src/styles/header.scss

```diff
@@ -0,0 +1,63 @@
+$blue: #00acdc;
+$main-accent: #060b35;
+$secondary: #616070;
```

Comment:
you use these variables in 2 files already, put them in separate file and use everywhere where you need them